index.bs

```diff
@@ -183,9 +183,13 @@ Integration with HTML {#html}
 
 This specification integrates with the [[!HTML]] specification by patching the algorithms below:
 
-Navigation response {#navigation-response}
+Speculative HTML parsing {#speculative-html-parsing}
 ----------
+At [=speculative fetch=], after the sub-step for the meta element "viewport" step add a new sub-step with the following:
```

Comment:
"speculative fetch" doesn't link at preview. What definition is it support to link to?

Reply:
I meant to link to https://html.spec.whatwg.org/multipage/parsing.html#speculative-html-parsing